src/promptflow/tests/sdk_cli_test/unittests/test_config.py

```diff
@@ -4,12 +4,13 @@
 from pathlib import Path
 
 import pytest
-from azure.ai.ml.constants._common import AZUREML_RESOURCE_PROVIDER, RESOURCE_ID_FORMAT
 
 from promptflow._sdk._configuration import ConfigFileNotFound, Configuration
 from promptflow._utils.context_utils import _change_working_dir
 
 CONFIG_DATA_ROOT = Path(__file__).parent.parent.parent / "test_configs" / "configs"
+AZUREML_RESOURCE_PROVIDER = "Microsoft.MachineLearningServices"
```

Comment:
is this added exclusively for recording? I do see these more than once, shall we move them to a centralize place?

Reply:
if it comes from merge main, can ignore this comment